examples/simpleSend.ts

```diff
@@ -4,15 +4,19 @@ dotenv.config();
 
 const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
 const path = process.env.QUEUE_NAME || "";
+const numberOfMessages: number = parseInt((process.argv[2] || "1") as string);
```

Comment:
Why not keep this coming from the environment as well?

Reply:
Had that initially. However when testing different scenarios, I found it cumbersome to change the value of the environment variable. Hence came up with this approach of providing the value as a command line arg. 

If consistency is of higher priority then I can switch back to making it an environment variable.

Reply:
I think it makes sense to keep this like you have it -- as this is a sample it makes sense that you would want to pass around different numbers of messages to put in the queue without using environment variables.